lua/fff/rust/lib.rs

```diff
@@ -224,6 +255,10 @@ fn create_exports(lua: &Lua) -> LuaResult<LuaTable> {
         "refresh_git_status",
         lua.create_function(refresh_git_status)?,
     )?;
+    exports.set(
```

Comment:
I don't think we need to expose it to the lua API, this is more of an internal API